bft/testers/postgres/runner.py

```diff
@@ -125,6 +126,10 @@ def run_sql_case(self, case: Case, mapping: SqlMapping) -> SqlCaseResult:
                 if len(arg_names) != 3:
                     raise Exception(f"Between function with {len(arg_names)} args")
                 expr = f"SELECT {arg_names[0]} BETWEEN {arg_names[1]} AND {arg_names[2]} FROM my_table;"
+            elif mapping.local_name == 'count(*)':
+                if len(arg_names) < 1:
```

Comment:
Is this reversed?  Are you only concerned with getting arguments when count * implies none?

Reply:
oops...i was just copying those exceptions parts from elsewhere.  I'll remove them